.github/workflows/coverage.yml

```diff
@@ -0,0 +1,56 @@
+name: Code coverage
+on:
+  pull_request:
+  push:
+    branches:
+      - TDE_REL_17_STABLE
+
+jobs:
+  collect:
+    name: Collect and upload
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Clone repository
+        uses: actions/checkout@v4
+        with:
+          submodules: recursive
+
+      - name: Install dependencies
+        run: ci_scripts/ubuntu-deps.sh
+
+      - name: Build postgres
+        run: ci_scripts/make-build.sh debug --enable-coverage
```

Comment:
Shouldn't this also have a `--continue` flag? Otherwise a failing test will also result in a lower coverage, because the test stops early.

Reply:
I assume you talking about `make-test`, not `make-build`. If some test failed coverage report would not be uploaded at all (and should not). As I understood `--keep-going` option in `make` is needed for situation where we have target `A` that depends on targets `B` and `C`, and if `B` failed with `keep-going` option make still will execute `C`.  If that's how it work then it's redundant here as we run only pg-tde tests here, am I wrong?